src/components/uielements/filterMenu/FilterMenu.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { bn } from '@thorchain/asgardex-util'
+
+import { getTickerFormat } from '../../../helpers/stringHelper'
+import { AssetPair } from '../../../types/asgardex.d'
+import CoinData from '../coins/coinData'
+import FilterMenu from './FilterMenu'
+
+const filterFunction = (item: AssetPair, searchTerm: string) => {
+  const tokenName = getTickerFormat(item.asset)
+  if (tokenName === null) return false
```

Comment:
In current implementation of `getTickerFormat` `tokenName` can't be `null`, but an empty string.

Reply:
Oops, I was wrong ^ . In BEPSwap we never had a `null` `tokenName`, but it has been changed in https://github.com/thorchain/asgardex-electron/blob/5ea3d57620f114dbbc64d7927a7ba6518ed3c481/src/helpers/stringHelper.ts#L8-L12 

Nevermind my previous 
comment!